.github/workflows/python-package.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: "ubuntu-22.04"
     strategy:
       matrix:
-        python-version: ["3.8", "3.9"]
+        python-version: ["3.9", "3.10"]
```

Comment:
Why not pythons up to 3.13?

Reply:
added up to 13 here https://github.com/lvgig/model_interpreter/pull/2/commits/43e2f03a3cae4ecd9aeb80018808d654c67ac8a0

but 13 had a mysterious error that i suggest we spinout into another bug/issue, so updated to just go up to 12
https://github.com/lvgig/model_interpreter/pull/2/commits/ad1bc53c9f3afafeaf4cd0b7ef567535d24a6f40

Reply:
Logged in #6